src/rf/rf/settings/base.py

```diff
@@ -277,15 +275,15 @@ def get_env_setting(setting):
 WSGI_APPLICATION = '%s.wsgi.application' % SITE_NAME
 # END WSGI CONFIGURATION
 
-# AWS CONFIGURATION
 
+# AWS CONFIGURATION
 AWS_BUCKET_NAME = environ.get('RF_S3_BUCKET', 'raster-foundry-test-uploads-67dc48c70b3bcf89eab78dbf5cf7900')  # NOQA
-AWS_PROFILE = environ.get('AWS_PROFILE', 'rf-dev')
```

Comment:
I think that the references in `views.py` can be removed because Boto will automatically detect `AWS_PROFILE`. Fixed in https://github.com/azavea/raster-foundry/commit/b8f63fc5632b5936e325e4dbab279af080db550b.